lib/guide.md

```diff
@@ -0,0 +1,1061 @@
+import com.swmansion.starknet.provider.rpc.JsonRpcProvider;
+
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Collections;
+import java.util.List;
+
+public class Main {
+    public static void main(String[] args) throws IOException, InterruptedException {
+        // Create a provider for interacting with Starknet
+        Provider provider = new JsonRpcProvider("https://example-node-url.com/rpc");
+
+        // Set up an account
+        Felt privateKey = Felt.fromHex("0x1234");
+        Felt publicKey = StarknetCurve.getPublicKey(privateKey);
+        Felt accountAddress = Felt.fromHex("0x1236789");
+        // ⚠️ WARNING ⚠️ Both the account address and private key are for demonstration purposes only.
+
+        StarknetChainId chainId = provider.getChainId().send();
+        Account account = new StandardAccount(accountAddress, privateKey, provider, chainId, Felt.ZERO);
+
+        // Import a compiled contract
+        Path contractPath = Paths.get("contract.sierra.json");
+        Path casmPath = Paths.get("contract.casm.json");
+        String contractCode = String.join("", Files.readAllLines(contractPath));
+        String casmCode = String.join("", Files.readAllLines(casmPath));
+
+        Cairo1ContractDefinition contractDefinition = new Cairo1ContractDefinition(contractCode);
+        CasmContractDefinition casmContractDefinition = new CasmContractDefinition(casmCode);
+        Felt nonce = account.getNonce().send();
+
+        DeclareTransactionV2Payload payload = account.signDeclareV2(
+                contractDefinition,
+                casmContractDefinition,
+                new ExecutionParams(nonce,Felt.ZERO),
+                true
+        );
+        Request<EstimateFeeResponseList> request = provider.getEstimateFee(List.of(payload), Collections.emptySet());
+        EstimateFeeResponseList response = request.send();
+        EstimateFeeResponse feeEstimate = response.getValues().get(0);
+        
+        System.out.println("The estimated fee is: " + feeEstimate.getOverallFee().getValue() + ".");
+    }
+}
+```
+
+# Package com.swmansion.starknet.account
+Account interface used to simplify preparing, signing Starknet transactions and automatic fee estimation.
+
+
+## Example usage of `StandardAccount`
+<!-- codeSection(path="starknet/account/StandardAccountTest.kt", function="exampleAccountUsage", language="kotlin") -->
+
+# Package com.swmansion.starknet.crypto
+
+Cryptography and signature related classes.
+This is a low level module. Recommended way of using is through
+Signer and Account implementations.
+
+# Package com.swmansion.starknet.data
+Data classes representing Starknet objects and utilities for handling them.
+
+# Package com.swmansion.starknet.data.types
+Data classes representing Starknet objects.
+
+# Package com.swmansion.starknet.data.types.transactions
+Data classes representing Starknet transactions.
```

Comment:
From what I see this package has been removed and transactions are now part of `com.swmansion.starknet.data.types`

Reply:
Good catch, I edited description of `com.swmansion.starknet.data.types` as follows:
```
# Package com.swmansion.starknet.data.types
Data classes representing Starknet objects and transactions.
```